protos/follow_me/follow_me.proto

```diff
@@ -30,19 +30,32 @@ service FollowMeService {
 
 // Configuration type.
 message Config {
-    // Direction relative to the target that the vehicle should follow
+    // Direction relative to the target that the vehicle should follow. Measured in reference to target's course (where it's moving towards)
     enum FollowDirection {
-        FOLLOW_DIRECTION_NONE = 0; // Do not follow
-        FOLLOW_DIRECTION_BEHIND = 1; // Follow from behind
-        FOLLOW_DIRECTION_FRONT = 2; // Follow from front
-        FOLLOW_DIRECTION_FRONT_RIGHT = 3; // Follow from front right
-        FOLLOW_DIRECTION_FRONT_LEFT = 4; // Follow from front left
+        FOLLOW_DIRECTION_BEHIND = 0; // Follow from behind
+        FOLLOW_DIRECTION_FRONT = 1; // Follow from front
+        FOLLOW_DIRECTION_FRONT_RIGHT = 2; // Follow from front right
+        FOLLOW_DIRECTION_FRONT_LEFT = 3; // Follow from front left
+        FOLLOW_DIRECTION_RIGHT = 4; // Follow from right
+        FOLLOW_DIRECTION_LEFT = 5; // Follow from left
+        FOLLOW_DIRECTION_BEHIND_RIGHT = 6; // Follow from behind right
+        FOLLOW_DIRECTION_BEHIND_LEFT = 7; // Follow from behind left
+        FOLLOW_DIRECTION_NONE = 8; // Follow direction invalid (defaults to Behind)
     }
 
-    float min_height_m = 1 [(mavsdk.options.default_value)="8.0"]; // Minimum height for the vehicle in meters (recommended minimum 8 meters)
-    float follow_distance_m = 2 [(mavsdk.options.default_value)="8.0"]; // Distance from target for vehicle to follow in meters (recommended minimum 1 meter)
-    FollowDirection follow_direction = 3; // Direction to follow in
-    float responsiveness = 4 [(mavsdk.options.default_value)="0.5"]; // How responsive the vehicle is to the motion of the target (range 0.0 to 1.0)
+    // Altitude mode to configure which altitude the follow me will assume the target to be at.
+    enum FollowAltitudeMode {
+        FOLLOW_ALTITUDE_MODE_CONSTANT = 0; // Target assumed to be mobing at a constant altitude of home position (where the vehicle armed)
+        FOLLOW_ALTITUDE_MODE_TERRAIN = 1; // Target assumed to be at the terrain level sensed by the distance sensor
+        FOLLOW_ALTITUDE_MODE_TRACKTARGET = 2; // Target GPS altitude taken into account to do 3D tracking
```

Comment:
I would write it like this:

```suggestion
        FOLLOW_ALTITUDE_MODE_TRACK_TARGET = 2; // Target GPS altitude taken into account to do 3D tracking
```

So that it becomes `trackTarget` or `track_target` or `TRACK_TARGET` in the generated codes for the language bindings (mavsdk-python, mavsdk-java, etc).